apps/server/src/shared/repo/schoolexternaltool/school-external-tool.repo.integration.spec.ts

```diff
@@ -1,16 +1,25 @@
 import { EntityManager, ObjectId } from '@mikro-orm/mongodb';
 import { Test, TestingModule } from '@nestjs/testing';
-import { ExternalTool, type School, SchoolExternalTool } from '@shared/domain';
+import {
+	ExternalTool,
+	type School,
+	SchoolExternalTool,
+	SchoolExternalToolDO,
+	CustomParameterEntryDO,
+} from '@shared/domain';
 import { MongoMemoryDatabaseModule } from '@shared/infra/database';
 import { ExternalToolRepoMapper } from '@shared/repo/externaltool/external-tool.repo.mapper';
-import { cleanupCollections, externalToolFactory, schoolExternalToolFactory, schoolFactory } from '@shared/testing';
+import {
+	cleanupCollections,
+	externalToolFactory,
+	schoolExternalToolFactory,
+	schoolFactory,
+	schoolExternalToolDOFactory,
+} from '@shared/testing';
 import { Logger } from '@src/core/logger';
 import { createMock } from '@golevelup/ts-jest';
-import { SchoolExternalToolDO } from '@shared/domain/domainobject/external-tool/school-external-tool.do';
-import { CustomParameterEntryDO } from '@shared/domain/domainobject/external-tool/custom-parameter-entry.do';
-import { schoolExternalToolDOFactory } from '@shared/testing/factory/domainobject/school-external-tool.factory';
+import { SchoolExternalToolQuery } from '@src/modules/tool/uc/dto';
```

Comment:
That is not good, circular dependency.
You use DOs in tool module. Please move this type to domain

Reply:
This is a DTO not a DO.